<launch>
    <node pkg="door_pass" type="door_passing.py" name="door_passing"  output="screen"/>
    <node pkg="door_pass" type="door_checking.py" name="door_checking"  output="screen"/>
    <node pkg="door_pass" type="door_wait_and_pass.py" name="door_wait_and_pass"  output="screen">
        <param name="max_trans_vel" value="0.4"/>
        <param name="n_closed_door" value="15"/>
    </node>
    <node pkg="door_pass" type="door_wait_and_move_base.py" name="door_wait_and_move_base"  output="screen">
        <param name="n_closed_door" value="15"/>
    </node>
    <node pkg="door_pass" type="door_waiting.py" name="door_waiting"  output="screen"/>
    <node pkg="door_pass" type="door_prediction.py" name="door_prediction"  output="screen"/>
</launch>
